WPMedia/fusion-news-theme-blocks
# THEMES-758: Identity Block | Sign Up Block

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

This PR adds the sign up block, converted to the v2 architecture. The sign up block covers the bulk of the sign up page, the rest is handled by the login links and social sign on blocks that will be converted to v2 in the near future.

## Jira Ticket

- [THEMES-758](https://arcpublishing.atlassian.net/browse/THEMES-758)

## Acceptance Criteria

The Identity Sign Up - Arc Block is using the v2 architecture.

## Test Steps

1. Checkout this branch `git checkout THEMES-758`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block`
3. Create a test page using the Single Column layout.
4. Place the "Identity Sign Up - Arc Block" on the page.
5. Set the Terms and Conditions URL and Privacy Policy URL values for the block.
6. Make sure the styling matches the designs in Figma (https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr/Enhanced-Styling-Design-System?type=design&node-id=8411-119293&mode=design&t=BeXCdtxm1ekWDLRJ-4).
7. The links in the terms and conditions and privacy text should match what is configured in the block.


## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-758]: https://arcpublishing.atlassian.net/browse/THEMES-758?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
